### DISCUSSION TITLE
Which version can be used on macOS 10.15?

### DISCUSSION THREAD

**Discussion by sylvaingml** [pyzo/pyzo#1076] [Q&A]
Hi,

I'm trying to ru pyzo on an old macOS 10.15 system (my daughter's computer).

I cannot run from Finder.

From Terminal, I tried "open /Applications/pyzo.app" and get this error:
> LSOpenURLsWithRole() failed with error -10810 for the file /Applications/pyzo.app.

Running directly the binary is more helpful: "/Applications/pyzo.app/Contents/MacOS/pyzo"
> Sorry, "pyzo" cannot be run on this version of macOS. Qt requires macOS 11.0.0 or later, you have macOS 10.15.7.

I have installed latest Python from binary distribution.

**What is the last version of Pyzo that supports 10.15?**

Thanks for your help!

**Comment by bdieterm** [comment]
I have no macOS computer, so take the following advice with a grain of salt:

> Qt requires macOS 11.0.0 or later

This is because Qt6 needs at least macOS 11:
https://doc.qt.io/qt-6/supported-platforms.html

But Qt5 still runs on older macOS versions >= 10.13:
https://doc.qt.io/qt-5/supported-platforms.html

> What is the last version of Pyzo that supports 10.15?

The last binary version available here at GitHub is probably the 32-bit version of Pyzo 4.11.7 because it is the last one shipped with Qt5.
https://github.com/pyzo/pyzo/releases/download/v4.11.7/pyzo-4.11.7-macos.dmg


I rather recommend to run Pyzo from source:
macOS 10.15 "Catalina" has a system Python v2.7, so you need to install a newer Python, for example the Miniconda distribution:

Miniconda System requirements: macOS 10.13+
https://docs.anaconda.com/miniconda/miniconda-system-requirements/

In the new Python interpreter from Miniconda, install PyQt5 and then run Pyzo from source as described in
https://github.com/pyzo/pyzo/tree/main#running-pyzo-from-source
You can then use the Miniconda Python also as the Python for the shell.
And you could also use pyinstaller to build your own binary of Pyzo if you prefer that over executing Pyzo from source.

**Reply by sylvaingml** [reply]
Ok. You confirmed what I feared. 
I will try those options. 
Thanks
